src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -141,12 +143,15 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
           </div>
         </DialogContent>
         <DialogActions>
-          <Button color="secondary" variant="outlined" form="activate-work-package-form" onClick={onHide}>
-            Cancel
-          </Button>
-          <Button color="success" variant="contained" type="submit" form="activate-work-package-form">
-            Submit
-          </Button>
+          <Box textAlign="left" sx={{ my: 2 }}>
+            <NERFailButton variant="outlined" form="activate-work-package-form" onClick={onHide} sx={{ mx: 2 }}>
```

Comment:
mx should be 1 and not 2 here. Because you have 1 on both, it adds up to 2px of space b/w the buttons

Reply:
just changed it back to mx = 1